R/flea_starters.R

```diff
@@ -46,25 +48,40 @@ ff_starters.flea_conn <- function(conn, week = 1:17, ...) {
     ) %>%
     tidyr::pivot_longer(c("home", "away"), names_to = "franchise", values_to = "player") %>%
     tidyr::hoist("player", "proPlayer", "owner", "points" = "viewingActualPoints") %>%
-    tidyr::hoist("proPlayer",
+    tidyr::hoist(
+      "proPlayer",
       "player_id" = "id",
       "player_name" = "nameFull",
       "pos" = "position",
-      "team" = "proTeamAbbreviation"
+      "team" = "proTeamAbbreviation",
+      "injury" = "injury"
+    ) %>%
+    tidyr::hoist(
+      "injury",
+      "injury_type" = "typeFull",
+      "injury_desc" = "description",
+      "injury_severity" = "severity"
```

Comment:
I think hoist is a bit safer than unnest_wider in my experience, have switched to that over the bind approach